packages/api/controllers/nomination.js

```diff
@@ -77,7 +79,13 @@ const updateNomination = async (req, res) => {
     //depending on status of application
     //current nominations don't have decline status, that should come after nominations hit ready for board review. TBD
     if (nomination.changed('status') && nomination.status === 'Decline') {
-      sendDeclineEmail(updatedNom)
+      sendDeclineEmail(nomination)
+    }
+    
+    // if nomination state is 'Document Review' then call the sendSurveyEmail
+    if (nomination.changed('status') && nomination.status === 'Document Review') {
```

Comment:
We can restructure these checks so `nomination.changed('status')` is only checked once instead of twice.

Reply:
That is a great point, I looked over the code and the nomination.changed('status') is a Boolean, if we intend to collect specific events we should target the nomination.status current state.